src/agent/src/rpc.rs

```diff
@@ -519,10 +519,7 @@ impl protocols::agent_ttrpc::AgentService for agentService {
         req: protocols::agent::CreateContainerRequest,
     ) -> ttrpc::Result<Empty> {
         match self.do_create_container(req) {
-            Err(e) => Err(ttrpc::Error::RpcStatus(ttrpc::get_status(
-                ttrpc::Code::INTERNAL,
-                e.to_string(),
-            ))),
+            Err(e) => Err(ttrpc_rpc_error(ttrpc::Code::INTERNAL, e.to_string())),
```

Comment:
How about we simplify even further by creating a macro so all this would become something like:

```rust
Err(e) => ttrpc_error!(e),
```
That way, we can tweak the implementation more easily in future too.

Reply:
Thanks @jodh-intel , I will do some update, and ttrpc_internal_error!(e) is a better name.

Reply:
@jodh-intel After consideration I tend to keep the code unchanged. 

The ttrpc_internal_error! is not a good name either. because there is a lot error types and we can't define macros for every type.

I prefer function to macro, what about `ttrpc_error(e)`?

Reply:
I prefer function to macro, what about `ttrpc_error(e)`?

Reply:
I have updated the name to ttrpc_error(e) already.